diff --git a/dist/module.mjs b/dist/module.mjs
index 5fd93a7a47968b58114c0056b8749f3118a72b07..987184c7dfbd387403ec0245d325051baa11d887 100644
--- a/dist/module.mjs
+++ b/dist/module.mjs
@@ -102,23 +102,6 @@ const module = defineNuxtModule({
     });
     addImportsDir(resolve("./runtime/composables"));
     addTemplates(options, nuxt);
-    const theme$1 = theme || { env: "NUXT_UI_PRO_LICENSE", link: "https://ui.nuxt.com/pro" };
-    const key = process.env[theme$1.env] || nuxt.options.uiPro?.license;
-    if (nuxt.options.dev || nuxt.options._prepare || nuxt.options.test) {
-      if (nuxt.options.dev && !key) {
-        consola.box(
-          colors.greenBright("Nuxt UI Pro") + `
-
-Missing \`${theme$1.env}\` env variable, please add it to your \`.env\`.
-
-` + colors.blueBright(`Purchase Nuxt UI Pro at ${theme$1.link} to build your app in production.`)
-        );
-      }
-      return;
-    }
-    nuxt.hook("build:before", async () => {
-      await validateLicense({ key, theme: theme$1, dir: nuxt.options.rootDir });
-    });
   }
 });
 
